.circleci/config.yml

```diff
@@ -21,20 +21,34 @@ executors:
       LEIN_ROOT: "true"   # we intended to run lein as root
       JVM_OPTS: -Xmx3200m # limit the maximum heap size to prevent out of memory errors
     <<: *defaults
+
+  # JDK 8 is preferred for deployments (https://github.com/benedekfazekas/mranderson/issues/57)
+  # Parallelism is disabled, at least for now (https://github.com/benedekfazekas/mranderson/issues/56)
+  openjdk8_deploy:
+    docker:
+      - image: circleci/clojure:openjdk-8-lein-2.9.1-node
```

Comment:
Are there no images with 2.9.3? Not a big deal, just looks a bit weird to use a different version of Lein for different JDKs.

Reply:
Actually it seems that 2.9.5 is the newest.

Reply:
I think that with `-node` it only goes up to 2.9.1, for this jdk version

And probably `-node` is needed, as it certainly is for the test suite and maybe the mrandersoning?

Reply:
I see. I guess down the road we can just install whatever else is needed ourselves in a more stripped down container, so that the setup in more consistent.